### PR TITLE
Update Japanese styleguides

### PR DESCRIPTION
**Pull request by marsf**
Merge latest Japanese styleguides from mozilla-japan repository.

https://github.com/mozilla-japan/translation/issues/260
https://github.com/mozilla-japan/translation/issues/463
https://github.com/mozilla-japan/translation/issues/519

### CODE REVIEW

**Review by peiying2 — APPROVED** [review]

**Comment by marsf** [comment]
Thanks!

**Comment by flodolo** [comment]
Please don't land PRs with linter issues. Could you fix them?
https://github.com/mozilla-l10n/styleguides/runs/2494266633

File: docs/ja/editorialguideline.md [hunk 1]
```diff
@@ -1,155 +1,173 @@
+# ガイドラインの適用範囲
```
Comment (flodolo) [inline docs/ja/editorialguideline.md:1]: The first line (h1) should be the title of the page, i.e. the equivalent of "Editorial Guidelines".

All the other title should move down one level (h1 -> h2, h2 -> h3).

File: docs/ja/editorialguideline.md [hunk 2]
```diff
@@ -1,155 +1,173 @@
...
+その他の翻訳記事等では、製品やサイトの表記に従ってください。もちろんそれらへの強制力はありませんが、ガイドラインとして活用していただけると、Mozilla 関連文書の統一が取れ、文書の品質を上げる助けになります。
+
 # 表記ルール
 
-mozilla.jp の表記ルールとして採用しているものを公開しています。mozilla.jp に限らず各サイトでの表記の参考にしてください。
+mozilla.jp の表記ルールとして採用していたものを公開しています。mozilla.jp に限らず各サイトでの表記の参考にしてください。
 
 ## 英数字の表記
 
-* 特に理由がなければ半角を用いる
```
Comment (flodolo) [inline docs/ja/editorialguideline.md:7]: While GitHub's Markdown flavor allows for both, we use asterisks for list items, and that's enforced by a linter that is now failing badly.

**Pull request merged**